pallets/api/src/fungibles/mod.rs

```diff
@@ -95,13 +95,12 @@ pub mod pallet {
 	#[pallet::pallet]
 	pub struct Pallet<T>(_);
 
+	/// The events that can be emitted.
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
 		/// Event emitted when allowance by `owner` to `spender` changes.
 		Approval {
-			/// Token ID.
```

Comment:
Sadly I think this is problematic. I understand it was probably removed to conform to the standard, but does ommitting it effectively make it useless? The events are emitted from a pallet managing multiple assets, so without knowing which asset id had an approval, is there any point? A contract call might be able to correlate the resulting events based on the extrinsic events, but if contract makes two calls to the fungibles pallet to create an allowance, for two different assets, how can we tell from the events which value matches which asset?

Reply:
But perhaps this brings us back to this [comment](https://github.com/r0gue-io/pop-node/pull/153#pullrequestreview-2229006536). 

In short, reason for adding events to the fungible pallet:
- standard compliance - the problem is that the events are not useful though (see frank's comment). In addition, when we do add the `asset_id` the events become the same as in pallet assets (e.g. [Transferred](https://github.com/paritytech/polkadot-sdk/blob/819a5818284a96a5a5bd65ce67e69bab860d4534/substrate/frame/assets/src/lib.rs#L563))
- pallet usage - we have an indicator how often the pallet is used. More interesting is how often the API is used

Would it be oke to start without events in the pallet?

Reply:
> Would it be oke to start without events in the pallet?

I would rather we start with something, even super simple events so we can be aware of if the API is being used on testnet.

Reply:
Alright I will then add the `asset id` back in the `Transfer` and `Approval` event